api/mapping/views.py

```diff
@@ -32,42 +31,63 @@
 from django.views import generic
 from django.views.decorators.csrf import csrf_protect
 from django.views.decorators.debug import sensitive_post_parameters
-from django.views.generic import ListView, DetailView
-from django.views.generic.edit import (CreateView, DeleteView, FormView,
-                                       UpdateView)
+from django.views.generic import DetailView, ListView
+from django.views.generic.edit import CreateView, DeleteView, FormView, UpdateView
 from extra_views import ModelFormSetView
 
-from .forms import (DictionarySelectForm, DocumentFileForm, DocumentForm,
-                    ScanReportAssertionForm, ScanReportForm, UserCreateForm,
-                    NLPForm)
-from .models import (DataDictionary, Document, DocumentFile, OmopField,
-                     OmopTable, ScanReport, ScanReportAssertion,
-                     ScanReportField, ScanReportTable, ScanReportValue,
-                     StructuralMappingRule, NLPModel)
+from .forms import (
+    DictionarySelectForm,
+    DocumentFileForm,
+    DocumentForm,
+    NLPForm,
+    ScanReportAssertionForm,
+    ScanReportForm,
+    UserCreateForm,
+)
+from .models import (
+    DataDictionary,
+    Document,
+    DocumentFile,
+    NLPModel,
+    OmopField,
+    OmopTable,
+    ScanReport,
+    ScanReportAssertion,
+    ScanReportField,
+    ScanReportTable,
+    ScanReportValue,
+    StructuralMappingRule,
+)
 from .services import process_scan_report, run_usagi
```

Comment:
Do we need run_usagi?

Reply:
Good question. Depends if you think we'll want to use the Usagi code in the future. I'd be happy to delete it and never see it again :D